DependencyInjection/Compiler/NotificationCompilerPass.php

```diff
@@ -50,14 +51,25 @@ public function process(ContainerBuilder $container)
 
                 $informations[$event['type']][] = $id;
 
-                $definition->addMethodCall(
-                    'addListener',
-                    array(
-                        $event['type'],
-                        array(new ServiceClosureArgument(new Reference($id)), 'process'),
-                        $priority,
-                    )
-                );
+                if (Kernel::MAJOR_VERSION <= 3 && Kernel::MINOR_VERSION < 3) {
```

Comment:
Please read this OP : https://symfony.com/doc/current/setup/bundles.html#updating-your-code-to-support-symfony-2-x-and-3-x-at-the-same-time